.drone.star

```diff
@@ -0,0 +1,195 @@
+config = {
+	'app': 'owncloud-sdk',
+
+	'branches': [
+		'master'
+	],
+
+	'yarnlint': True,
+
+	'build': True
+}
+
+def main(ctx):
+    return [ fullBuildAndPublish(), testWithinSubFolder(), publish() ]
+
+def incrementVersion():
+    return [{
+    		'name': 'increment-version',
+    		'image': 'node:11',
+    		'pull': 'always',
+    		'commands': [
+    			'yarn version --no-git-tag-version --new-version 1.0.0-${DRONE_BUILD_NUMBER}'
+    		],
+            'when': {
+                'event': [
+                    'push'
+                ]
+            }
+    }]
+
+def buildDocs()	:
+    return [{
+    		'name': 'build-docs',
+    		'image': 'node:11',
+    		'pull': 'always',
+    		'commands': [
+    			'yarn install',
+    			'yarn build:docs'
+    		],
+    }]
+
+def buildSystem():
+    return [{
+    		'name': 'build-system',
+    		'image': 'node:11',
+    		'pull': 'always',
+    		'commands': [
+    			'yarn install',
+    			'yarn lint',
+    			'yarn build:docs'
+    		],
+    }]
+
+def prepareTestConfig(subFolderPath = '/'):
+    return [{
+        'name': 'prepare-test-config',
+        'image': 'owncloud/ubuntu:16.04',
+		'commands': [
+			'apt update',
+			'apt install gettext -y',
+			'envsubst < tests/config/config.drone.json > tests/config/config.json',
+			'cat tests/config/config.json'
+		],
+        'environment' : {
+            'SUBFOLDER': subFolderPath
+        }
+    }]
+
+def test():
+    return [{
+        'name': 'test',
+        'image': 'owncloudci/chromium',
+        'pull': 'always',
+        'commands': [
+            'yarn test-drone'
+        ]
+    }]
+
+def publishDocs():
+    return [{
+        'name': 'publish-docs',
+        'image': 'plugins/gh-pages:1',
+        'pull': 'always',
+        'settings': {
+            'pages_directory': 'docs'
+        },
+        'environment' : {
+            'GITHUB_PASSWORD': {
+                'from_secret': 'github_password'
+            },
+            'GITHUB_USERNAME': {
+                'from_secret': 'github_username'
+            }
+        },
+        'when': {
+            'event': [
+                'push'
+            ]
+        }
+    }]
+
+def publishSystem():
+    return [{
+        'name': 'publish-system',
+        'image': 'plugins/npm:1',
+        'pull': 'always',
+        'environment' : {
+            'NPM_EMAIL': {
+                'from_secret': 'npm_email'
+            },
+            'NPM_TOKEN': {
+                'from_secret': 'npm_token'
+            },
+            'NPM_USERNAME': {
+                'from_secret': 'npm_username'
+            },
+        },
+        'when': {
+            'event': [
+                'push'
+            ]
+        }
+    }]
+
+def fullBuildAndPublish():
+    return {
+        'kind': 'pipeline',
+        'name': 'Full build and publish',
```

Comment:
this pipeline does not publish anymore

Reply:
That will only run after merge so we need to merge this Pr to see if that works.